packages/common/pipes/file/parse-file.pipe.ts

```diff
@@ -39,26 +39,23 @@ export class ParseFilePipe implements PipeTransform<any> {
   }
 
   async transform(value: any): Promise<any> {
-    if (this.thereAreNoFilesIn(value)) {
-      if (this.fileIsRequired) {
-        throw this.exceptionFactory('File is required');
-      }
-      return value;
-    }
+    const areThereNoFileIn = this.thereAreNoFilesIn(value);
 
-    if (this.validators.length) {
-      if (Array.isArray(value)) {
-        await this.validateFiles(value);
-      } else {
-        await this.validate(value);
-      }
-    }
+    if (areThereNoFileIn && this.fileIsRequired)
+      throw this.exceptionFactory('File is required');
+
+    if (!areThereNoFileIn && this.validators.length)
+      await this.validateFilesOrFile(value);
```

Comment:
To remain consistent with the rest of the codebase
```suggestion
    if (areThereNoFileIn && this.fileIsRequired) {
      throw this.exceptionFactory('File is required');
    }
    if (!areThereNoFileIn && this.validators.length) {
      await this.validateFilesOrFile(value);
    }
```

Reply:
thank you for reviewing my code :)